source/common/network/listen_socket_impl.h

```diff
@@ -80,6 +80,28 @@ template <typename T> class NetworkListenSocket : public ListenSocketImpl {
 
   Socket::Type socketType() const override { return T::type; }
 
+  // These four overrides are introduced to perform check. A null io handle is possible only if the
+  // io handle is listener side.
```

Comment:
What does "is listener side" mean? Clarify?

Reply:
Updated to 
```
A null io handle is possible only if the the owner socket is a listen socket that does not bind to port.
```

Is it better?